actionview/lib/action_view/routing_url_for.rb

```diff
@@ -147,5 +135,9 @@ def optimize_routes_generation? #:nodoc:
       def _generate_paths_by_default
         true
       end
+
+      def only_path_for(host)
```

Comment:
@kaspth Though this method name seems good considering it returns boolean value for now, but what if in future _generate_paths_by_default returns some other value than boolean?
What do you think?
Should we still go with only_path?